docs/api/utils.md

```diff
@@ -106,6 +106,57 @@ const ls = Bun.which("ls", {
 console.log(ls); // null
 ```
 
+You can think of this as a builtin alternative to the [`which`](https://www.npmjs.com/package/which) npm package.
+
+## `Bun.randomUUIDv7()`
+
+`Bun.randomUUIDv7()` returns a [UUID v7](https://www.ietf.org/archive/id/draft-peabody-dispatch-new-uuid-format-01.html#name-uuidv7-layout-and-bit-order), which is monotonic and suitable for sorting and databases.
+
+```ts
+import { randomUUIDv7 } from "bun";
+
+const id = randomUUIDv7();
+// => "0192ce11-26d5-7dc3-9305-1426de888c5a"
+```
+
+A UUID v7 is a 128-bit value that encodes the current timestamp, a random value, and a counter. The timestamp is encoded using the lowest 48 bits, and the random value and counter are encoded using the remaining bits.
+
```

Comment:
maybe also add something like: the counter ensures UUIDs generated with the same timestamp have the expected sequencing